api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst

```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
 |                          |                         |``ENABLED``. The default |
 |                          |                         |value is ``ENABLED``.    |
 +--------------------------+-------------------------+-------------------------+
+|cipherProfile             |String                   |Specifies a cipher       |
+|                          |                         |profile to be used. The  |
+|                          |                         |default value is         |
+|                          |                         |``default``.             |
++--------------------------+-------------------------+-------------------------+
+
+.. note::
+
+   CipherProfile's are defined by the service and are immutable.
+   The service currently offers two profiles. The 'default' profile, which
+   is set on any load balancer enabling SSLTermination by default. The second
+   profile that can optionally be selected is ``CLBCipherPolicy2017-08``.
+   :ref:`Ciphers <ciphers>` section for more details about available profiles.
+
+.. note::
+
+   In order to remove a ``cipherProfile`` set ``cipherProfile``
```

Comment:
In order to remove a ``cipherProfile``, set

Reply:
Removed this entire section.